rebasehelper/config.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you to rebase package to the latest version
+# Copyright (C) 2013-2014 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#          Tomas Hozza <thozza@redhat.com>
+
+import os.path
+
+from six.moves import configparser
+
+from rebasehelper.logger import logger
+from rebasehelper.options import OPTIONS, traverse
+
+
+class Conf:
+    def __init__(self, path_to_conf):
+        self.path_to_config = self.get_conf_path(path_to_conf)
+        self.config = self.get_config()
+
+    def get_conf_path(self, path_to_conf):
+        default_options = ("$XDG_CONFIG_HOME/rebase-helper.cfg", "$HOME/.config/rebase-helper.cfg")
+
+        if path_to_conf is None:
+            if os.path.isfile(default_options[0]):
```

Comment:
This won't work, you have to expand the environment variables first. You could use [`os.path.expandvars`](https://docs.python.org/2/library/os.path.html#os.path.expandvars) for that, but I think better approach would be to expand `XDG_CONFIG_HOME` first, defaulting to `$HOME/.config`, and then use the expanded value.

Try this for example:
```python
path = os.environ.get('XDG_CONFIG_HOME', os.path.expandvars('$HOME/.config'))
config_file = os.path.join(path, 'rebase-helper.cfg')
```

Reply:
Also, the name of the config file should be defined as a constant somewhere, e.g. in `rebasehelper/constants.py` or `rebasehelper/settings.py` (BTW these two files should be probably merged together, but that's not in scope of this PR).